### ISSUE TITLE
[v6] Support custom meta props?

### ISSUE THREAD

**Issue by smirea** [erikras/redux-form#1634]
Use-Case: apart from the existing flags (dirty, touched, ...) i'd like to implement my own custom state `highlighted` to globally highlight fields based on interaction. 

I've tried using a separate `highlighted: {}` state in the form, but mimicking the tree anywhere else is clumsy. Changing the value of the input is also not ideal (aka from `foo: String` to `foo: {value: String, highlighted: Bool}`) since that requires changing the signature of the model, which, more often than not, is based on a database object

Ideally the interaction would be:

```
combineReducers: {
    form: form.plugin((state, action) => {
        if (action.type !== 'highlight') return state;
            return {
                    ...state,
                    fields: {
                        ...state.fields,
                        [action.meta.field]: {
                            ...state.fields[action.meta.field],
                            highlighted: action.value,
                        },
                    },
                };
    })
}
```

and in my component:

```
<Field name='foo' component={render} />

render = ({input, meta: {highlighted}}) => 
     <div style={highlighted ? 'fancy' : null}><input {...input} /></div>
```

but that's not available since the meta props are hand-picked and hardcoded.

**TL; DR:** what do you think of adding `...state` on line https://github.com/erikras/redux-form/blob/master/src/createFieldProps.js#L68 to enable user defined meta props

**Comment by erikras** [comment]
Fix published in [`v6.0.3`](https://github.com/erikras/redux-form/releases/tag/v6.0.3).
